Test/SchemaCompilerSpec.hs

```diff
@@ -460,4 +460,4 @@ getInstanceDecl instanceName full =
         takeInstanceDecl (line:rest)
             | isEmpty line = []
             | otherwise = line : takeInstanceDecl rest
-        takeInstanceDecl [] = error "never encountered newline?"
+        takeInstanceDecl [] = [] -- EOF reached
```

Comment:
I would like to draw attention to this change here

In the test cases the instance I am looking for is the last one in the generated code, which resulted in an error, even tho i think it's totally fine. That this results in an error looked like a simple oversight to me so I changed it - the tests still pass... However a second opinion would be great in cause this actually had a deeper reason.

Reply:
Looks good to me 👍